HTML/Currents.html

```diff
@@ -0,0 +1,133 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <title>Currents — Keep Talking and Nobody Explodes Module</title>
+    <meta name="viewport" content="initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src="js/ktane-utils.js"></script>
+</head>
+<body>
+        <div class="section">
+            <div class="page page-bg-01">
```

Comment:
Spacing here is off now that you've deleted that line, it'll need to be readjusted throughout the document.

Reply:
H O W
how i didnt remove it. It gotta be one of these cursed prs.